ref_ts/owslib_revised/waterml/wml.py

```diff
@@ -0,0 +1,937 @@
+from __future__ import (absolute_import, division, print_function)
```

Comment:
@zhiyuli So now we're maintaining or own fork of OWSLib?  Who is going to track security updates and apply them over time?  OWSLib is open source, and the developer is good about taking contributions (I have done so).  We should use a specific version of OWSLib and contribute necessary changes if the current version is broken for wml.